### ISSUE TITLE
CVE Services 2.x suppport

### ISSUE THREAD

**Issue by gwd** [wizedkyle/cveservices-go-sdk#4]
Hello,

You said here [1] that you were going to move to CVE Services 2.x support, using `swagger-codegen`, at some point; it looks like that hasn't happened yet? 

Let me know if there's anything I can do to help.

[1] https://github.com/CVEProject/cve-services/issues/681#issuecomment-1135223965

**Comment by gwd** [comment]
Hey!  Just wanted to ping to see if any progress had been made; or if anything could be done to help.

**Comment by wizedkyle** [comment]
@gwd Sorry for letting this go stale I got caught up in a few other projects over the past few months.

I plan to pick this back up starting in May when I have a bit more capacity.

**Comment by gwd** [comment]
No worries -- my plan was, "Wait for a month or two and then generate the bindings myself", but I haven't managed to find the time either.  Nothing like a deadline to focus the mind, however, and our current automation (around the github pull requests) will stop working at the end of June, if I'm remembering right. :-)

**Comment by wizedkyle** [comment]
I know that feeling 😂

I'm pretty sure you are spot on with the end of June should be plenty of time to get this up to scratch.

**Comment by gwd** [comment]
BTW I've run `openapi-generator` and put the resulting code here: https://gitlab.com/xen-project/misc/golang-cveapi .  Haven't used it yet, other than to run `go test -v ./...`.  Happy to have you copy the repo over wholesale, or replace it with your copy once you've got it up.

**Comment by wizedkyle** [comment]
@gwd I should have a new version up in the next week or so just working through some of the testing.

I am also going to commit the new version over here: https://github.com/ForgeResponse/cveservices-go-sdk just as that relates to a project I am working on at the moment.

**Comment by gwd** [comment]
Hmm, looks like the generated bindings don't actually work.  Have you run into this?

https://github.com/CVEProject/cve-services/issues/1074

**Comment by wizedkyle** [comment]
@gwd Yep I am running into the same issue as well, I am reluctant to manually create the models as that will become painful to maintain over time.
